### DISCUSSION TITLE
Multiple crawlers in different folders

### DISCUSSION THREAD

**Discussion by asafeclemente** [dadoonet/fscrawler#1380] [Q&A] [answered]
I have several folders and each one I need to put in an elasticsearch index, for that I need I need to instantiate several fscralwler services, but due to the amount, I can't run them all at the same time. Is there any way to automatically shut down when a crawler finishes indexing documents? Or some smarter way to do this process?

**Comment by dadoonet** [comment, marked as answer]
Yes. You can use `-loop 1` option. It will run once and exit.

see https://fscrawler.readthedocs.io/en/fscrawler-2.9/user/options.html

**Reply by asafeclemente** [reply]
that's good, thank you!!
